src/angular-sql-query.js

```diff
@@ -257,9 +318,15 @@
         });
     }
 
-    function removeQueryBackUp(params) {
+    /**
+     * ???
```

Comment:
This one is to be able to remove a resource with another params than just the id.